preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Setting.java

```diff
@@ -334,21 +336,21 @@ public F getField() {
   }
 
   /**
-   * Saves the current value of this setting using a {@link StorageHandler}.
+   * Saves the current value of this setting using a {@link StorageHandlerImpl}.
    *
-   * @param storageHandler the {@link StorageHandler} to use
+   * @param storageHandler the {@link StorageHandlerImpl} to use
    */
   public void saveSettingValue(StorageHandler storageHandler) {
     storageHandler.saveObject(getBreadcrumb(), value.getValue());
   }
 
   /**
-   * Loads the value of this setting using a {@link StorageHandler}.
+   * Loads the value of this setting using a {@link StorageHandlerImpl}.
```

Comment:
should still be "StorageHandler", since if an own implementation of the `StorageHandler` is being used, it may not be called `StorageHandlerImpl`